addons/project/views/project_views.xml

```diff
@@ -921,6 +921,13 @@
                                         <span attrs="{'invisible': ['|', ('repeat_show_week', '=', False), ('repeat_show_month', '=', False)]}">of</span>
                                         <field name="repeat_month" attrs="{'invisible': [('repeat_show_month', '=', False)], 'required': [('repeat_show_month', '=', True)]}" />
                                     </div>
+                                    <field name="mon" invisible="1"/>
```

Comment:
We could add a comment to explain why it is needed. 🙂 
 ```suggestion
                                    <!-- Those fields are added to trigger the compute method for the recurrence feature. -->
                                    <field name="mon" invisible="1"/>
```

Reply:
@xavierbol , Done!!